package.json

```diff
@@ -20,38 +20,37 @@
     "clean": "rimraf api.md coverage/"
   },
   "dependencies": {
-    "caniuse-lite": "^1.0.30001038",
-    "electron-to-chromium": "^1.3.390",
+    "caniuse-lite": "^1.0.30001040",
+    "electron-to-chromium": "^1.3.402",
     "node-releases": "^1.1.53",
-    "pkg-up": "^2.0.0"
+    "pkg-up": "^3.1.0"
   },
   "bin": "./cli.js",
   "devDependencies": {
-    "@logux/eslint-config": "^35.0.4",
-    "@logux/sharec-config": "^0.5.7",
-    "@size-limit/preset-small-lib": "^4.4.2",
-    "clean-publish": "^1.1.6",
-    "cross-spawn": "^7.0.1",
+    "@logux/eslint-config": "^36.1.2",
```

Comment:
You forget to run `yarn test`. My new ESLint config force ES6 syntax. This update broke `yarn test`.